src/rotest/core/result/handlers/stream/dots_handler.py

```diff
@@ -1,10 +1,30 @@
 """"Stream dots layout result handler."""
 # pylint: disable=invalid-name,too-few-public-methods,arguments-differ
 # pylint: disable=too-many-arguments,super-init-not-called,unused-argument
+from functools import wraps
+
 from base_handler import BaseStreamHandler
+from rotest.core.flow_component import AbstractFlowComponent
 from rotest.common.constants import GREEN, YELLOW, RED, BOLD, CYAN, BLUE
 
 
+def check_main_decorator(func):
+    """Only call the handler method on test cases and top test flows.
```

Comment:
documentation is pretty obscure, it just ignores everything that is within a flow.

Reply:
Given the proposed decorator name above, this documentation actually completes the whole picture and gives us some more positive explanation about the action of this decorator